lib/cfme/cloud_services/inventory_sync.rb

```diff
@@ -29,6 +29,27 @@ def self.sync_queue(userid, targets)
         MiqTask.generic_action_with_callback(task_opts, queue_opts)
       end
 
+      def self.collect(manifest:, targets:, task_id: nil, _miq_task_id: nil)
```

Comment:
TODO shouldn't need _both_ task_id and miq_task_id

https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_queue.rb#L562
https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_task.rb#L315

We should probably update https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_queue.rb#L562 to set `params[:task_id]`